website/events/templates/events/email/registration_queued_email.txt

```diff
@@ -0,0 +1,8 @@
+Dear {{ registration.member.profile.display_name|safe }},
```

Comment:
It might be good to add a closing to the txt emails too (the html parent template adds it automatically, but txt has no parent template.